docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
-```code
-# Once Alice logs back in, she will be able to view audit logs
-$ tctl users update alice --set-roles=editor,reviewer,auditor
-```
-</TabItem>
+See the [Machine ID introduction](../../enroll-resources/machine-id/introduction.mdx) for more information.
+
+## Teleport roles
 
-</Tabs>
+Some examples of roles include:
 
-Because Alice has two or more roles, permissions from those roles create a union. She
-will be able to act as a system administrator and auditor at the same time.
+- **access**: Grants basic access to resources.
+- **editor**: Allows users to modify configuration settings.
+- **auditor**: Allows users to view logs and auditing activities.
 
-## Step 2/3. Map SSO users to roles
+Roles enforce Role-Based Access Control (RBAC), ensuring that users only have the permissions needed for their specific tasks. 
+See [Teleport Access Controls Reference](../../reference/access-controls/roles.mdx) for in-depth information about types of roles, options, and usage.
 
-Next, follow the instructions to set up an authentication connector that maps
-users within your SSO solution to Teleport roles.
+## Understanding Teleport’s Resource Tagging and Matching
 
-### Teleport Enterprise
+Before configuring RBAC, it’s essential to understand how Teleport tags and matches resources to users’ roles.
 
-Create a SAML or OIDC application that Teleport can integrate with, then
-create an authentication connector that maps users within your application to
-Teleport roles.
+- Resource Tagging: Teleport uses labels (key-value pairs) to tag resources like SSH nodes, databases, or Kubernetes clusters. For example, a server might have the label env: production.
+- Role Matching: A role in Teleport allows or denies access to resources based on matching these labels. For instance, if a user has a role that allows access to resources labeled env: staging, they will only access resources with that label.
 
-<Tabs>
-<TabItem label="SAML">
+You can use this labeling and matching logic to grant users specific access, ensuring they only interact with the resources they need.
 
-Follow our [SAML Okta Guide](./sso/okta.mdx) to
-create a SAML application.
+## Understanding Teleport’s YAML Configuration
 
-Save the file below as `okta.yaml` and update the `acs` field.
-Any member in Okta group `okta-admin` will assume a built-in role `admin`.
+Teleport’s roles are defined using YAML. Here’s a breakdown of a basic role configuration:
+
+- **kind**: The type of resource being created. For roles, this will always be role.
+- **metadata**: Includes information like the role’s name, which is used to identify the role.
+- **spec**: This section defines what the role can and cannot do.
+- **allow**: Lists the permissions the role grants, such as allowed logins or access to resources based on labels.
+- **deny**: Specifies any permissions that should be explicitly denied, even if they would otherwise be allowed.
+
+##  Creating a Basic Role for SSH Access
```

Comment:
I would make the guide more focused as a how-to guide or a conceptual guide, rather than bringing together features of both of them (see the [style guide](https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#organizing-information-into-purpose-driven-documents)).